Sources/Tools/plNetLog/ChunkBuffer.h

```diff
@@ -50,6 +50,12 @@ You can contact Cyan Worlds, Inc. by email legal@cyan.com
 #include <QString>
 #include <QMutex>
 
+#if QT_VERSION >= QT_VERSION_CHECK(6, 0, 0)
+using Q_MUTEX_LOCKER_TYPE = QMutexLocker<QMutex>;
+#else
+using Q_MUTEX_LOCKER_TYPE = QMutexLocker;
+#endif
```

Comment:
This should probably look more like a type than a define (eg not be all caps and/or snake case).